utils/cryptoTransaction.js

```diff
@@ -0,0 +1,34 @@
+function isBalanceSufficient (senderCoins, reveicerCoins) {
+  if (senderCoins.copper >= reveicerCoins.copper && senderCoins.silver >= reveicerCoins.silver && senderCoins.gold >= reveicerCoins.gold) {
+    return true
+  }
+  return false
+}
+
+function updateBalance (payer, receiver, requestCoins) {
+  const payerCoins = {
+    copper: diff(payer.copper, requestCoins.copper),
+    silver: diff(payer.silver, requestCoins.silver),
+    gold: diff(payer.gold, requestCoins.gold)
+  }
+  const receiverCoins = {
+    copper: add(receiver.copper, requestCoins.copper),
+    silver: add(receiver.silver, requestCoins.silver),
+    gold: add(receiver.gold, requestCoins.gold)
+
+  }
+  return { payerCoins, receiverCoins }
+}
+
+function diff (a, b) {
+  return (a - b)
+}
+
+function add (a, b) {
+  return (a + b)
+}
```

Comment:
Why not use a arrow function, all can come in one line itself
```javascript
const add = (a, b) => a + b
```

same goes with `diff` function